### DISCUSSION TITLE
Is setVoice customisable? 

### DISCUSSION THREAD

**Discussion by thealmamun** [dlutton/flutter_tts#507] [Q&A]
I found a line in example like await flutterTts.setVoice({"name": "Karen", "locale": "en-AU"}); from where the name 'Karen' comes from? is it customisable? I mean is there any other names and voice out there?

**Comment by echogit** [comment]
Yes =, there are many others, that's is how voices are classified on iOS (Iphone/Ipad). On iOS there are better voices (enhanced) that you can download to use in your TTS app.

**Reply by Lukiya** [reply]
Where can I download enhanced voices?

**Reply by echogit** [reply]
The voices are already on the configuration, but you need to go to the settings in order to download them.

**Reply by Lukiya** [reply]
I managed to change the voice, great, thanks for the tips.
